Intersect.Client/Interface/Game/DescriptionWindows/Components/RowContainerComponent.cs

```diff
@@ -0,0 +1,109 @@
+﻿using Intersect.Client.Framework.Gwen.Control;
+
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Intersect.Client.Interface.Game.DescriptionWindows.Components
+{
+    public class RowContainerComponent : ComponentBase
+    {
+        protected KeyvalueRowComponent mKeyValueRow;
+
+        protected JObject KeyValueRowLayout;
+
+        protected int mComponentY = 0;
+
+        public RowContainerComponent(Base parent, string name) : base(parent, name)
+        {
+            GenerateComponents();
+            LoadLayout();
+            GetLayoutTemplates();
+            DestroyLayoutComponents();
+        }
+
+        protected override void GenerateComponents()
+        {
+            base.GenerateComponents();
+
+            mKeyValueRow = new KeyvalueRowComponent(mContainer);
+        }
+
+        protected void GetLayoutTemplates()
+        {
+            KeyValueRowLayout = mKeyValueRow.GetJson();
+        }
+
+        protected void DestroyLayoutComponents()
+        {
+            mKeyValueRow.Dispose();
+        }
+
+        protected void PositionComponent(ComponentBase component)
+        {
+            component.SetPosition(component.X, mComponentY);
+            mComponentY += component.Height;
+        }
+
+        public override void CorrectWidth()
+        {
+            base.CorrectWidth();
+            var margins = mContainer.Margin;
+
+            mContainer.SetSize(mContainer.Width, mContainer.Height + margins.Bottom);
+        }
+
+        public KeyvalueRowComponent AddKeyValueRow(string key, string value)
+        {
+            var row = new KeyvalueRowComponent(mContainer, key, value);
```

Comment:
Key**V**alueRowComponent seems to got a naming case problem i guess?

Reply:
Yeah, probably a typo from when I was messing about with it. Fixed that now though, thank you. :)